src/mixins/common-mixin.ts

```diff
@@ -38,16 +38,15 @@ export default class CommonMixin extends Vue {
   async validateAndScroll (flags: object, components: object): Promise<boolean> {
     // Create an array of the _ordered_ validity flags
     const validFlagArray = Object.keys(flags).map(key => flags[key])
-
     // Find the _first_ corresponding component that is invalid
     const component = document.getElementById(components[validFlagArray.indexOf(false)])
 
     // If there is an invalid component, scroll to it
     if (component) {
```

Comment:
What was happening before is there would be an incomplete component on a filing for example take a specialResolution with an invalid component certify, since flags-review-certify-interface.ts was general the order
would set  validFlagArray would contain a false component however it would set the detail component as the false one then this line would not evaluate to true as there is no detail component on a specialResolution. So true would be returned.

Reply:
I'm still thinking that it's OK for un-rendered components to be in the array as long as their value is true (valid), which is how the array is initialized in state-model.ts.

The question is, which un-rendered component is invalid? Maybe we should fix that problem instead.

(Is there another issue with the components being in different orders? That shouldn't matter much.)